### ISSUE TITLE
Accessibility finding: Cards aren't properly contained in a list

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#137241]
### Checklist item

WEB-131-004	Lists use proper list formatting

### Severity level

2, High. Should be fixed in 1-2 sprints post-launch.

### Page(s)/url(s)

https://staging.va.gov/my-va

### Details

Lists of cards aren't contained in semantic lists. According to VADS guidance: "Use a `ul` for a card group and an `li` for each card. This formatting allows screen readers to enumerate the items in the card group and allows shortcuts between list items."

### Reproduction steps

Inspect the code of the cards.

### Proposed solution or next steps

Follow VADS guidance - put each card group in a `ul` and each card in an `li`.
